src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -122,11 +122,13 @@
     <GenerateStaticWebAssetEndpointsPropsFile
       StaticWebAssets="@(V4Assets)"
       StaticWebAssetEndpoints="@(V4AssetEndpoints)"
+      PackagePathPrefix="staticwebassets/V4"
```

Comment:
Do we know what changed to break this? Is this a sign of some underlying bug?

Reply:
I don't know yet, but I plan to look at it on Monday, at a quick glance, the endpoints in the package are incorrect too (we will have to service that).

It doesn't currently fail I think because we have a process to "upgrade" older packages and define endpoints and the endpoints end up being generated as part of the user build.

Not sure what the difference is between the build in the repo and the end user build that causes it to fail here. In the end, it's a bug in the package.

Reply:
Filed an issue to track